test/transforms_test.dart

```diff
@@ -23,20 +23,26 @@ void main() {
     final res = await postgrest
         .from('users')
         .select()
-        .order('catchphrase', ascending: true)
+        .order('status', ascending: true)
         .order('username')
         .execute();
     expect(
-      ((res.data as List)[0] as Map)['username'],
-      'kiwicopple',
-    );
-    expect(
-      ((res.data as List)[2] as Map)['username'],
-      'supabot',
+      (res.data as List).map((row) => (row as Map)['status']),
+      [
+        'ONLINE',
+        'ONLINE',
+        'OFFLINE',
+        'OFFLINE',
+      ],
     );
     expect(
-      ((res.data as List)[3] as Map)['username'],
-      'dragarcia',
+      (res.data as List).map((row) => (row as Map)['username']),
+      [
+        'supabot',
+        'awailas',
+        'kiwicopple',
+        'dragarcia',
+      ],
     );
```

Comment:
The values here do not seem like the correct one at first glance, because other tests are modifying the values in test table. Probably will fix the tests so that each test case will reset the table after execution in later PR.